tests/Unit/ConfigMgrCBDsc.ReverseDsc.tests.ps1

```diff
@@ -5912,6 +6112,66 @@ InModuleScope $script:subModuleName {
                 SiteCode             = 'Lab'
                 PSComputerName       = 'localhost'
             }
+
+            $siteConfig = @{
+                SiteCode = 'Lab'
+                Include  = 'SiteConfiguration'
+            }
+
+            $invokeSiteConfigurationPri = @{
+                SiteCode                                          = 'Lab'
+                Comment                                           = 'Site Lab'
+                ClientComputerCommunicationType                   = 'HttpsOrHttp'
+                ClientCheckCertificateRevocationListForSiteSystem = $true
+                UsePkiClientCertificate                           = $false
+                UseSmsGeneratedCert                               = $true
+                RequireSigning                                    = $true
+                RequireSha256                                     = $false
+                UseEncryption                                     = $false
+                MaximumConcurrentSendingForAllSite                = 6
+                MaximumConcurrentSendingForPerSite                = 3
+                RetryNumberForConcurrentSending                   = 2
+                ConcurrentSendingDelayBeforeRetryingMins          = 10
+                EnableLowFreeSpaceAlert                           = $true
+                FreeSpaceThresholdWarningGB                       = 10
+                FreeSpaceThresholdCriticalGB                      = 5
+                ThresholdOfSelectCollectionByDefault              = 100
+                ThresholdOfSelectCollectionMax                    = 1000
+                SiteSystemCollectionBehavior                      = 'Warn'
+                SiteType                                          = 'Primary'
+                EnableWakeOnLan                                   = $true
+                WakeOnLanTransmissionMethodType                   = 'Unicast'
+                RetryNumberOfSendingWakeupPacketTransmission      = 1
+                SendingWakeupPacketTransmissionDelayMins          = 10000
+                MaximumNumberOfSendingWakeupPacketBeforePausing   = 10
+                SendingWakeupPacketBeforePausingWaitSec           = 3
+                ThreadNumberOfSendingWakeupPacket                 = 10
+                SendingWakeupPacketTransmissionOffsetMins         = 10
+                ClientCertificateCustomStoreName                  = 'SMSStore'
+                TakeActionForMultipleCertificateMatchCriteria     = 'SelectCertificateWithLongestValidityPeriod'
+                ClientCertificateSelectionCriteriaType            = 'ClientAuthentication'
+                ClientCertificateSelectionCriteriaValue           = 'Personal'
```

Comment:
missing Configurationname, DependsOn, ModuleName, and several other settings if this is being returned by Invoke-DSCResource not required but would be uniform settings and would be returned by the invoke.  They are in the $invokeSiteConfigurationCas

Reply:
Fixed in local repo.